samples/meeting-app/csharp/README.md

```diff
@@ -0,0 +1,148 @@
+﻿## MeetingApp (HR recruitment) Sample
+
+This sample illustrates a meeting experience for HR recruitment.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple bot that accepts input from the user and echoes it back.
```

Comment:
Could you please update the description?

Reply:
changed, please check.